CONTRIBUTING.md

```diff
@@ -275,13 +311,9 @@ programs that customers could have been writing against the current version
 of the CDK, that will no longer "work correctly" with the proposed new
 version of the CDK.
 
-Breaking changes are not allowed in *stable* libraries¹. They are permissible
-but still *highly discouraged* in experimental libraries, and require explicit
-callouts in the bodies of Pull Requests that introduce them.
-
-> ¹) Note that starting in version 2 of the CDK, the majority of library code will be
```

Comment:
Elad explicitly requested I was super explicit about this, which is why it's here.

Do you think it's not worthwhile? (I mean what the section says is sorta implied by the section before it, but again, explicitness...)